opencti-platform/opencti-front/src/private/components/data/playbooks/PlaybookAddComponents.jsx

```diff
@@ -458,6 +459,18 @@ const PlaybookAddComponentsContent = ({
                       />
                     );
                   }
+                  if (k === 'organizations') {
+                    return (
+                      <ObjectOrganizationField
+                        key={k}
+                        name="organizations"
+                        style={{ marginTop: 20, width: '100%' }}
+                        label={'Target organizations'}
```

Comment:
There is no i18n needed here ?

Reply:
nothing is translated in playbook components for now, so I didn't work on it in this fix. you can see just above the same thing for authorized members field with label "Targets"